
KeyPoint[] keypoints
Pose3D pose
sensor_msgs/Image image
bool target_detected
geometry_msgs/Point[] target_points # corners and center
